djstripe/views.py

```diff
@@ -51,7 +51,7 @@ def get_context_data(self, *args, **kwargs):
             context['subscription'] = customer.current_subscription
         except CurrentSubscription.DoesNotExist:
             context['subscription'] = None
```

Comment:
I think a lot of this method is already in a subscription mixin. If there's more (useful) functionality in this method, move it to mixins.py and make this view inherit that mixin.